src/clic-subcommand-instance.adb

```diff
@@ -561,25 +639,33 @@ package body CLIC.Subcommand.Instance is
             Error_Exit (1);
          end if;
 
-         --  Initialize a new switch parser that will only see the new
-         --  sub-command line (i.e. the remaining args and switches after
-         --  global parsing).
-         Initialize_Option_Scan (Parser, Sub_Cmd_Line);
+         if not Cmd.Switches_As_Args then
 
-         --  Parse sub-command line, invalid switches will raise an exception
-         Getopt (Command_Config.GNAT_Cfg, Parser => Parser);
+            --  Skip sub-command switch parsing as requested by the command
+            Sub_Arguments := Global_Arguments;
+         else
```

Comment:
The `if` logic seems reversed here, although I'm sure it's me not following through some detail. Just in case I'm leaving this comment.

Reply:
Nevermind, my mistake.